src/Content/css/responsive-style.css

```diff
@@ -0,0 +1,255 @@
+@media (max-width: 700px) {
+	.burger-btn {
+		position: absolute;
+    display: block;
+    float: left;
+  	margin: 13px 13px;
+  	cursor: pointer;
+  }
+
+	.nav-logo {
+		width: 125px;
+    height: 41px;
+		display: inline-block;
+		overflow-y: hidden;
+	}
+
+  .nav-logo a{
+    background: url("../img/small-logo.png");
+  }
+  .nav-logo a:hover{
+    background: url("../img/small-logo.png");
+  }
+  .nav-list{
+    display: none;
+  }
+	.navigation	{
+		height: 50px;
+		position: relative;
+		text-align: center;
+		border: 0;
+		display: block;
+	}
+
+	.fiture-certification{
+		display: none;
+	}
+
+	.meeting-title{
+	  display: inline-block;
+	  font-size: 40px;
+	  margin-top: 0;
+	  padding: 0 20px;
+	  color: #f1f1f1;
+	}
+
+	.sidebar {
+		display: block;
+		position: fixed;
+		top: 0;
+		left: -270px;
+		width: 270px;
+		height: 100%;
+	}
+	.sidebar.active {
+		-webkit-transform: translateX(270px);
+		-webkit-transition: all .25s;
+	}
+
+	.navigation.active{
+		-webkit-transform: translateX(270px);
+		-webkit-transition: all .25s;
+	}
+
+	.content.active{
+		-webkit-transform: translateX(270px);
+		-webkit-transition: all .25s;
+		 position: fixed;
+	}
+
+	.mobile-nav-search{
+		height: 50px;
+		width: 270px;
+		margin-left: -40px;
+		border-bottom: 1px solid #474441;
+		box-shadow: inset -1px 0 2px rgba(0,0,0,.3);
+	}
+
+	.mobile-nav-list-item{
+		display: block;
+		height: 46px;
+		width: 270px;
+		margin-left: -40px;
+		border-bottom: 1px solid #474441;
+	}
+
+	.mobile-nav-list-item-active{
+		background-color: #262321;
+	}
+
+	.mobile-nav-list-item > a{
+		padding: 15px;
+		display: block;
+		line-height: 15px;
+		color: #f1f1f1;
+		text-decoration: none;
+	}
+
+	.mobile-nav-list-item > a:hover{
+		color: #f1f1f1;
+		text-decoration: none;
+	}
+
+	.mobile-nav-list-item > a > i{
+		display: block;
+		line-height: 15px;
+		color: #9d9b99;
+		float: right;
+	}
+
+	.mobile-input{
+		background-color: transparent;
+		border: none;
+		color: #f1f1f1;
+		box-shadow: none;
+		margin: 1px 0 0 2px;
+		width: 90%;
+		padding-left: 6px;
+		outline: none;
+	}
+
+	.search-container{
+		margin: -15px 10px 0 10px;
+		position: relative;
+		height: 30px;
+		background-color: #464340;
+	}
+
+	.modile-search-icon{
+		position: absolute;
+		right: 0;
+		margin: 5px 0 0 0;
+		background: transparent;
+		border: none;
+		color: #9d9b99;
+	}
+
+	.search-box{
+		display: none;
+	}
+
+	.adv-block{
+	  padding: 0 0;
+	}
+
+	.feature-row:first-child{
+		margin-top: 60px;
+	}
+
+	.feature-row{
+		margin: 7% 0;
+	}
+
+	.feature-row:last-child{
+		margin-bottom: 60px;
+	}
+
+	.meeting-title{
+		padding: 4px 19px;
+		line-height: 36px;
+		font-size: 30px;
+		text-align: left;
+		width: 100%;
+	}
+
+	.meeting-info{
+		padding: 12px 19px;
+		font-size: 18px;
+		line-height: 24px;
+		text-align: left;
+		width: 100%;
+	}
+
+	.meeting-block{
+		width: 89%;
+		margin: -20px auto 0 auto;
+	}
+}
+
+.sidebar{
```

Comment:
Styles for this class are presented few times. Why?

Reply:
Please also review all defined classes

Reply:
reviewed